sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -2808,7 +2728,11 @@ def get_words(self, **kwargs):  # pylint: disable=unused-argument
         :return: iterable[DocumentWord]
         :rtype: iterable[DocumentWord]
         """
-        return _get_children(self, search_elements=["word"], cross_page=False)
+        result = []
```

Comment:
do we need to add that error if someone does a to_dict/from_dict and the parent is not available?

Reply:
Yes, thanks for the reminder, will update this!

Reply:
Done, let me know if the error message is clear enough!